S2VX.Game/Editor/Containers/CommandPanel.cs

```diff
@@ -79,7 +85,59 @@ public class CommandPanel : OverlayContainer {
             }
         }
 
+        private void AddErrorIndicator() {
+            ErrorContainer.Add(new Box {
+                RelativePositionAxes = Axes.Both,
+                RelativeSizeAxes = Axes.Both,
+                Anchor = Anchor.TopCentre,
+                Origin = Anchor.Centre,
+                Colour = Color4.Red,
+                Width = 0.95f,
+                Height = 0.0025f,
+                Y = .0555f,
+                X = -.025f,
+            });
+            ErrorContainer.Add(new Box {
+                RelativePositionAxes = Axes.Both,
+                RelativeSizeAxes = Axes.Both,
+                Anchor = Anchor.TopCentre,
+                Origin = Anchor.Centre,
+                Colour = Color4.Red,
+                Width = 0.95f,
+                Height = 0.0025f,
+                Y = .095f,
+                X = -.025f,
+            });
+            ErrorContainer.Add(new Box {
+                RelativePositionAxes = Axes.Both,
+                RelativeSizeAxes = Axes.Both,
+                Anchor = Anchor.TopLeft,
+                Origin = Anchor.Centre,
+                Colour = Color4.Red,
+                Width = 0.0025f,
+                Height = 0.04f,
+                Y = .075f,
+                X = 0,
+            });
+            ErrorContainer.Add(new Box {
+                RelativePositionAxes = Axes.Both,
+                RelativeSizeAxes = Axes.Both,
+                Anchor = Anchor.TopRight,
+                Origin = Anchor.Centre,
+                Colour = Color4.Red,
+                Width = 0.0025f,
+                Height = 0.04f,
+                Y = .075f,
+                X = -.051f,
+            });
+        }
+
         private void HandleAddClick() {
```

Comment:
I would try catch this whole block and `Console.WriteLine` the Exception in the catch block. You've mentioned that you have to continue after getting an exception in `S2VXCommand.FromString` but this is fine.

Reply:
Why the entire function? Couldn't it just be around the S2VXCommand.FromString part?